terraform/modules/ecs_module/ecs.tf

```diff
@@ -0,0 +1,125 @@
+provider "aws" {
+  region = var.aws_region
+}
+
+resource "aws_ecs_cluster" "aws-ecs" {
```

Comment:
📝 **[tfsec]** <[AVD-AWS-0034](https://aquasecurity.github.io/tfsec/v1.27.6/checks/aws/ecs/enable-container-insight/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Cluster does not have container insights enabled.